plugins/azure/postgresqlserver/flexibleServerPrivateDns.js

```diff
@@ -0,0 +1,52 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Private DNS Zone Integrated',
+    category: 'PostgreSQL Server',
+    domain: 'Databases',
+    description: 'Ensure that PostgreSQL flexible servers has Private DNS Zone integrated.',
+    more_info: 'Integrate Private DNS Zones with PostgreSQL flexible servers to enhance DNS service reliability and security within your Azure virtual network, ensuring seamless DNS resolution and streamlined domain management.',
+    recommended_action: 'Ensures Vnet and Private DNS Zone (private access) is integrated for PostgreSQL flexible server.',
+    link: 'https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-networking-private',
```

Comment:
```suggestion
    link: 'https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-networking-private',
```
this is the same as vnet integration, can we add a link that mentions private dns zones?